pkg/stats/statsmgr.go

```diff
@@ -2,32 +2,62 @@ package stats
 
 import (
 	"fmt"
+	"sync"
 	"time"
 
 	"github.com/vesoft-inc/nebula-importer/pkg/base"
+	"github.com/vesoft-inc/nebula-importer/pkg/config"
+	"github.com/vesoft-inc/nebula-importer/pkg/csv"
 	"github.com/vesoft-inc/nebula-importer/pkg/logger"
 )
 
 type StatsMgr struct {
 	StatsCh       chan base.Stats
 	DoneCh        chan bool
-	NumFailed     int64
-	NumReadFailed int64
-	totalCount    int64
-	totalBatches  int64
-	totalLatency  int64
-	totalReqTime  int64
+	statsRW       sync.RWMutex
+	Stats         Stats
 }
 
-func NewStatsMgr(numReadingFiles int) *StatsMgr {
+type Stats struct {
+	NumFailed     int64 `json:"numFailed"`
+	NumReadFailed int64 `json:"numReadFailed"`
+	TotalLine     int64 `json:"totalLine"`
+	TotalCount    int64 `json:"totalCount"`
+	TotalBatches  int64 `json:"totalBatches"`
+	TotalLatency  int64 `json:"totalLatency"`
+	TotalReqTime  int64 `json:"totalReqTime"`
+}
+
+func NewStatsMgr(files []*config.File) *StatsMgr {
+	numReadingFiles := len(files)
+	totalLine := int64(0)
+	var wg sync.WaitGroup
+	wg.Add(numReadingFiles)
+	for _, file := range files {
+		path := file.Path
+		withHeader := file.CSV.WithHeader
+		go func(path string) {
+			lines, err := csv.CountLines(path, *withHeader)
```

Comment:
I think counting file lines is impractical for huge file in new coroutine. Why not to do this when reading these data files?